### DISCUSSION TITLE
blog/create-github-pages/

### DISCUSSION THREAD

**Discussion by giscus** [SunChulBaek/SunChulBaek.github.io#3] [Announcements]
# blog/create-github-pages/

요약    테마의 starter를 사용하면 깃허브 페이지를 쉽게 생성 할 수 있습니다.

https://sunchulbaek.github.io/blog/create-github-pages/

**Comment by SunChulBaek** [comment]
댓글 테스트

**Reply by SunChulBaek** [reply]
대댓글
